source/stylesheets/_slider.scss

```diff
@@ -0,0 +1,59 @@
+$slider-circle-color: #384467;
+$slider-circle-color-hover: #a4bfd1;
+$slider-circle-size: 20px;
+$slider-track-color: #a4bfd1;
+$slider-track-height: 10px;
+$slider-label-color: #384467;
+$slider-label-width: 60px;
+
+.slider {
+  width: 100%;
+  margin-bottom: 2em;
+
+  .slider__range {
+    width: 100%;
+    -webkit-appearance: none;
+    height: .25em;
+    background: $slider-track-color;
+    // TODO: determine better accessibility handling
+    // Potentially outline just the slider circle.
+    outline: none;
```

Comment:
Without this outline at all, it becomes impossible to tell when the user has the range selected so it's really hard to use with keyboard navigation. It also appears that the keyboard can't change the value because it's so slow to adjust that it makes it even more confusing since it looks like nothing's happening. I'd be down with the outline around the slider circle though since that would probably look nicer than an outline box around the whole range